### ISSUE TITLE
Notes twitching while in playback mode

### ISSUE THREAD

**Issue by IgorZev** [LyricWulf/Embers#89]
**Describe the bug**
When in playback mode the notes twitch up and down as they are falling.

**To Reproduce**
1. Select a song
2. Play it in playback mode
3. You'll see the notes twitching

**Expected behavior**
The notes smoothly fall down

**Desktop (please complete the following information):**
 - OS: Windows 10 Home Version 1903
 - Version 1.0.4

**Additional context**
I believe this could be fixed with VSYNC as it does seem that the problem really is just with the screen not showing the notes at the correct time

**Comment by LyricWulf** [comment]
Can you provide a video of this? I can't reproduce this issue.

**Comment by IgorZev** [comment]
I am actually unable to capture the bug because most of the screen recorders I used didn't have a fast enough recording framerate to capture it. I did use one recorder that should have captured it but funnily enough, it seemed to fix the problem (this was the recorder that came pre-installed in my machine and was specifically made for it).

I am actually not sure why this happens but it does seem to be an issue with the refresh rate of the screen isn't linked to the refresh rate of the program.

While testing out this bug I actually found 2 more but I'm not sure if they have been already addressed.

1. Sometimes notes can switch between the hand (blue and orange color)
Video: https://drive.google.com/open?id=13cvvQy7Bx2XztFrD0-JFMOK8nthxsa2o 
(the note in the middle changes hands, this was LyricWulf - Lone Digger.mid)

2. Changing it to 240fps seems to make it slower. I'm hesitant to say is lags because it's a little different from lagging. If it lagged it would be unstable, this looks like it just slows everything down. Have you used delta time in the program? If not this may be caused because of that
